<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageProcessing</class>
 <widget class="QDialog" name="ImageProcessing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1264</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>42</y>
     <width>651</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="hmin_label">
          <property name="text">
           <string>0    </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="smin_label">
          <property name="text">
           <string>0   </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="vmin_label">
          <property name="text">
           <string>0   </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QSlider" name="hMinSlider">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="pageStep">
             <number>10</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sMinSlider">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="pageStep">
             <number>10</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="vMinSlider">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="pageStep">
             <number>10</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QSlider" name="hMaxSlider">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="pageStep">
             <number>10</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sMaxSlider">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="pageStep">
             <number>10</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="vMaxSlider">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="pageStep">
             <number>10</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="hmax_label">
        <property name="text">
         <string>255</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="smax_label">
        <property name="text">
         <string>255</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="vmax_label">
        <property name="text">
         <string>255</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="startStream_pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Video Stream</string>
   </property>
  </widget>
  <widget class="QPushButton" name="endStream_pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>End Video Stream</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>30</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Erode Size</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>110</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Dilate Size</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>0</y>
     <width>101</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QDial" name="erode_dial">
        <property name="whatsThis">
         <string>set Erode Size</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="erode_label">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QDial" name="dilate_dial">
        <property name="whatsThis">
         <string>set Dilate Size</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="dilate_label">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Test Video</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>30</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>minArea</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="minArea_spinBox">
        <property name="maximum">
         <number>100000</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>maxArea</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="maxArea_spinbox">
        <property name="minimum">
         <number>100</number>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>hMinSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hmin_label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hMaxSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hmax_label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>836</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>868</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sMaxSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>smax_label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>836</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>868</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vMaxSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>vmax_label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>836</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>868</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sMinSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>smin_label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vMinSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>vmin_label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>464</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>erode_dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>erode_label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1024</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>1038</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dilate_dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dilate_label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1024</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>1038</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startStream_Clicked()</slot>
 </slots>
</ui>
